testnets/lumera/chain.json

```diff
@@ -0,0 +1,132 @@
+{
+  "$schema": "../../chain.schema.json",
+  "chain_name": "lumera",
```

Comment:
Please rename the chain_name to lumeratestnet, (and the directory name, as well)